docs/contributing.md

```diff
@@ -0,0 +1,22 @@
+Contribution Guide
+==================
+
+Licensing
+---------
+
+This project is licensed using the [BSD-3-Clause license](https://github.com/projectatomic/osbs-client/blob/master/LICENSE). When submitting pull requests please make sure your commit messages include a signed-off-by line. You can do this by using `git commit --signoff`.
+
+Submitting changes
+------------------
+
+Changes are accepted through pull requests.
+
+Please create your feature branch from the *master* branch. Make sure to add unit tests under the `tests/` subdirectory (we use py.test and flexmock for this). Tests are run automatically when you push new commits, but you can run them locally with `py.test tests` from the top directory.
```

Comment:
When you push new commits, tests will be triggered to be run in [travis-ci](https://travis-ci.org/projectatomic/osbs-client) and results will be shown in the PR. You can also run them...